apps/mobile/src/app/(home)/activity/components/activity-card.tsx

```diff
@@ -0,0 +1,49 @@
+import { ReactNode } from 'react';
+import { useAnimatedStyle, withSpring } from 'react-native-reanimated';
+
+import { Box, Pressable, PressableProps, usePressedState } from '@leather.io/ui/native';
+
+interface ActivityCardProps extends PressableProps {
+  icon: ReactNode;
+  children?: ReactNode;
+  onPress?(): void;
+  onLongPress?(): void;
+  testID?: string;
+  iconTestID?: string;
+}
+
+export function ActivityCard({ children, icon, onPress, ...props }: ActivityCardProps) {
+  const { pressed, onPressIn, onPressOut } = usePressedState(props);
+  const animatedStyle = useAnimatedStyle(() => ({
+    transform: [{ scale: withSpring(pressed ? 0.95 : 1) }],
```

Comment:
Our Pressable now accepts declarative presseEffect config. This can be replaced with:
```tsx
<Pressable         
  pressEffects={{
    opacity: { from: 1, to: 0.95 },
  }}
/>

```